app/controllers/idv/in_person/public/usps_locations_controller.rb

```diff
@@ -16,7 +16,7 @@ def index
             city: search_params['city'], state: search_params['state'],
             zip_code: search_params['zip_code']
           )
-          locations = proofer.request_facilities(candidate)
+          locations = proofer.request_facilities(candidate, nil)
```

Comment:
ditto re: switching to keyword arguments. And since `vector_of_trust` has a default, we can drop it here
```suggestion
          locations = proofer.request_facilities(location: candidate)
```